sdk/src/main/java/io/dapr/actors/runtime/ActorTypeInformation.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * The MIT License

```

Comment:
Update copyright

Reply:
I followed the copyright from the author of the file. Is it OK to change it as per the license terms?

Reply:
It should be our license like in any other file,

Reply:
This file is significantly different, updating license.